extractor/extraction_containers.cpp

```diff
@@ -217,21 +221,21 @@ void ExtractionContainers::PrepareEdges(lua_State *segment_state)
 
     while (edge_iterator != all_edges_list_end && node_iterator != all_nodes_list_end)
     {
-        if (edge_iterator->result.source < node_iterator->node_id)
+        if (edge_iterator->result.osm_source_id < node_iterator->node_id)
         {
             SimpleLogger().Write(LogLevel::logWARNING) << "Found invalid node reference " << edge_iterator->result.source;
             edge_iterator->result.source = SPECIAL_NODEID;
```

Comment:
Yeah, agreed SPECIAL is a strange choice.  To keep things consistent for this PR, I've renamed INVALID_OSM_NODEID to SPECIAL_OSM_NODEID.  This at least gets things consistent, and doesn't pollute this PR with a bunch of unrelated changes.  We can fix the SPECIAL/INVALID naming separately.